context-free.md

```diff
@@ -2,21 +2,28 @@
 
 ##  Who is this programming language for?
 
+I think this language is for people that want to make digital art using context-free grammars.
```

Comment:
For both of you, I think you could have added a little more detail. For example, I don't know that there are a lot of people who think "ooh I want to make art with context free grammars", but there are probably people who think "ooh, I really like fractals and these cool recursive patterns".